File: LeetCode Solutions/Problems/1266. Minimum Time Visiting All Points.cpp
class Solution {
public:
    int minTimeToVisitAllPoints(vector<vector<int>>& points) {
        int time = 0;
        for (int i=0; i<points.size()-1; i++){
            int x = points[i][0] - points[i+1][0];
            int y = points[i][1] - points[i+1][1];
            time += max(abs(x), abs(y));
        }
        return time;
    }
};

File: LeetCode Solutions/Problems/1266. Minimum Time Visiting All Points.py
class Solution:
    def minTimeToVisitAllPoints(self, points: List[List[int]]) -> int:
        time = 0
        for i in range(len(points)-1):
            x = points[i][0] - points[i+1][0]
            y = points[i][1] - points[i+1][1]
            time += max(abs(x), abs(y))
        return time